tm2/pkg/sdk/auth/ante.go

```diff
@@ -146,6 +146,9 @@ func NewAnteHandler(ak AccountKeeper, bank BankKeeperI, sigGasConsumer Signature
 
 			// check signature, return account with incremented nonce
 			sacc := signerAccs[i]
+			// Do not check signatures if the chainID is dev (gnodev) if we decide to verify the genesis signatures for
```

Comment:
I'd drop this comment entirely, it's very use-case specific and introduces more confusion than it needs to

Reply:
Droped: 09dce4322f68e8f7aa366c3c33da287d5761d8ae